assets/css/style.scss

```diff
@@ -0,0 +1,780 @@
+$font-stack: 'Red Hat Display', sans-serif;
+$color-dark: #28283D;
+$color-light-grey: #C4C4C4;
+$color-dark-green: #4D96A9;
+$color-dark-green-transparent: rgba(77, 150, 169,0.9);
+$color-light-green: #8FE3F9;
+$color-active-green: #71C0D4;
+$color-active-purple: #B18BDD;
+$color-white: white;
+$color-purple: #855FB1;
+$color-shadow: rgba(0, 0, 0, 0.25);
+$color-black: #000000;
+$color-grey: #87879D;
+$color-dirty-white: #FAFAFA;
+$color-light-purple: #D9B8FF;
+$color-line: #D8D8D8;
+$color-number: #87879D;
+
+@media only screen and (min-device-width: 375px) {
```

Comment:
Struktura mi se ne svidja. Tezimo da css svake konstante bude nestovan klasom kuju ima njen div. To bi znacilo da npr. imas body a unutar njega mediaQuery koji se odnosi na body. Pa tako redom za svaku klasu koju imas. A ne da nestujes po media query-ju, to nije optimalno i navodi te da ponavljas kod.

Reply:
U redu, na dalje cu tako pisati css.